src/KeyManager.sol

```diff
@@ -13,10 +14,14 @@ abstract contract KeyManagerBase {
     // Anyone can see the master public key
     address public xPub;
 
+    // placeholder for hardened and non_hardened child indexes
+    uint32 public constant HARDENED_START_INDEX = 2147483648;
```

Comment:
should probably be in KeyManager_v0
wouldn't it be easier to understand if it was `0x80000000` or 1<<31?